RedisGraph/RedisGraph
# skip aux when there are no graphs in keyspace

To test this do the following:

1. Run redis-server 7.2 or above with the Graph module
2. Create some NONE graph keys and SAVE
3. Reload the server **WITHOUT** the graph module and make sure the RDB is loaded
